views/beneficial-owner-type.html

```diff
@@ -73,6 +73,13 @@
   {% endif %}
 
   {% set beneficialOwnerOtherUrl = OE_CONFIGS.BENEFICIAL_OWNER_OTHER_URL %}
+
+  {% if FEATURE_FLAG_ENABLE_REDIS_REMOVAL %}
```

Comment:
we should have the one feature flag check and override the feature flag "off" values in that block.
so - {% set beneficialOwnerOtherUrl = OE_CONFIGS.BENEFICIAL_OWNER_OTHER_URL %} 
should go above the feature flag block and in the block override it with
{% set beneficialOwnerOtherUrl = beneficialOwnerOtherUrlWithParams %}

Reply:
Consolidated this.